### PR TITLE
Added quantity administered" as mandatory field

### PR DESCRIPTION
**Pull request by yash-egov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/checklist/checklist_eligibility_assessment.dart [hunk 1]
```diff
@@ -186,10 +186,10 @@ class _EligibilityChecklistViewPageState
                               ineligibilityReasons,
                             );
                             var descriptionText = ifIneligible
-                                ? '"Ineligible"'
+                                ? 'the beneficiary is ineligible for SPAQ administration'
                                 : ifReferral
-                                    ? '"Referral"'
-                                    : '"Administration"';
+                                    ? 'You are being redirected to the beneficiary referral screen'
+                                    : 'You are being redirected to SPAQ administration screen';
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/checklist/checklist_eligibility_assessment.dart]: @yash-egov @amitverma-egov why do we have hardcoded string why there is no lcoalisations

File: apps/health_campaign_field_worker_app/lib/widgets/beneficiary/resource_beneficiary_card.dart [hunk 2]
```diff
@@ -70,7 +79,7 @@ class _ResourceBeneficiaryCardState
                     formControlName: 'resourceDelivered.${widget.cardIndex}',
                     valueMapper: (value) {
                       return localizations.translate(
-                        value.sku ?? value.id,
+                        convertForLocalization(value.sku) ?? value.id,
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/beneficiary/resource_beneficiary_card.dart]: @amitverma-egov @yash-egov why do we need to convert what is the change we are trying here?? we can change the sku itself in the backend

Comment (yash-egov) [reply apps/health_campaign_field_worker_app/lib/widgets/beneficiary/resource_beneficiary_card.dart]: sku contains SPAQ-1 ( 3 a 11 messes) but we need to show only SPAQ-1

**Review by amitverma-egov — APPROVED** [review]
ok

**Pull request closed without merging**